File: years/2024/elixir/test/aoc/day_13_test.exs
defmodule AoC.Day13.Test do
  @moduledoc false

  use ExUnit.Case, async: false

  # comment this out to always log to the console
  @moduletag :capture_log

  describe "part 1" do
    test "with sample data" do
      assert AoC.Day13.part_1("../data/13.txt") == 0
    end
  end

  describe "part 2" do
    test "with sample data" do
      assert AoC.Day13.part_2("../data/13.txt") == 0
    end
  end
end

File: lang/clojure/skeleton/test/day_13_test.clj
(ns day-13-test
  (:require
   [clojure.test :refer :all]
   [day-13 :refer :all]))

(deftest day-13-test
  (testing "sample data"
    (is (= 0 (part1 "../data/13-sample.txt")))
    (is (= 0 (part2 "../data/13-sample.txt"))))
  (testing "puzzle data"
    (is (= 0 (part1 "../data/13.txt")))
    (is (= 0 (part2 "../data/13.txt")))))
